lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -63,6 +63,9 @@ function formatContentURL(content: URLContent) {
   if (content.url.startsWith('blackboard://')) {
     return 'PDF file in Blackboard';
   }
+  if (content.url.startsWith('canvas-studio://')) {
```

Comment:
We should make sure we always include content.name and be able to remove all of this.